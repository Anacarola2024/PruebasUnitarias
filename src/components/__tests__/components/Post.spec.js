import { mount } from '@vue/test-utils';
import { describe, it, expect } from 'vitest';
import PostView from '@/views/PostView.vue';

describe('Post.vue', () => {
  it('matches the snapshot', () => {
    const wrapper = mount(PostView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
