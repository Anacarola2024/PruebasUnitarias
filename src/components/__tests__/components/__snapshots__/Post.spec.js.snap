// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Post.vue > matches the snapshot 1`] = `
"<div class="posts">
  <h1>Posts</h1>
  <ul>
    <li></li>
    <li></li>
    <li></li>
    <li></li>
  </ul>
</div>"
`;
